integration_tests/e2e/refer/programmeHistory.cy.ts

```diff
@@ -283,121 +283,148 @@ context('Programme history', () => {
+          const programmeHistoryDetailsPage = Page.verifyOnPage(ProgrammeHistoryDetailsPage)
+          programmeHistoryDetailsPage.selectOutcome('complete')
+          programmeHistoryDetailsPage.inputOutcomeYearCompleted('not a number')
+          programmeHistoryDetailsPage.shouldContainButton('Continue').click()
+
+          const programmeHistoryDetailsPageWithError = Page.verifyOnPage(ProgrammeHistoryDetailsPage)
+          programmeHistoryDetailsPageWithError.shouldHaveErrors([
+            {
+              field: 'yearCompleted',
+              message: 'Enter a year using numbers only',
+            },
+          ])
         })
-        programmeHistoryPage.shouldContainSuccessMessage()
-      })
 
-      it('displays an error when inputting a non numeric value for `yearCompleted`', () => {
-        const programmeHistoryDetailsPage = Page.verifyOnPage(ProgrammeHistoryDetailsPage)
-        programmeHistoryDetailsPage.selectOutcome('complete')
-        programmeHistoryDetailsPage.inputOutcomeYearCompleted('not a number')
-        programmeHistoryDetailsPage.shouldContainButton('Continue').click()
-
-        const programmeHistoryDetailsPageWithError = Page.verifyOnPage(ProgrammeHistoryDetailsPage)
-        programmeHistoryDetailsPageWithError.shouldHaveErrors([
-          {
-            field: 'yearCompleted',
-            message: 'Enter a year using numbers only',
-          },
-        ])
+        it('displays an error when inputting a non numeric value for `yearStarted`', () => {
+          const programmeHistoryDetailsPage = Page.verifyOnPage(ProgrammeHistoryDetailsPage)
+          programmeHistoryDetailsPage.selectOutcome('incomplete')
+          programmeHistoryDetailsPage.inputOutcomeYearStarted('not a number')
+          programmeHistoryDetailsPage.shouldContainButton('Continue').click()
+
+          const programmeHistoryDetailsPageWithError = Page.verifyOnPage(ProgrammeHistoryDetailsPage)
+          programmeHistoryDetailsPageWithError.shouldHaveErrors([
+            {
+              field: 'yearStarted',
+              message: 'Enter a year using numbers only',
+            },
+          ])
+        })
       })
 
-      it('displays an error when inputting a non numeric value for `yearStarted`', () => {
-        const programmeHistoryDetailsPage = Page.verifyOnPage(ProgrammeHistoryDetailsPage)
-        programmeHistoryDetailsPage.selectOutcome('incomplete')
-        programmeHistoryDetailsPage.inputOutcomeYearStarted('not a number')
-        programmeHistoryDetailsPage.shouldContainButton('Continue').click()
-
-        const programmeHistoryDetailsPageWithError = Page.verifyOnPage(ProgrammeHistoryDetailsPage)
-        programmeHistoryDetailsPageWithError.shouldHaveErrors([
-          {
-            field: 'yearStarted',
-            message: 'Enter a year using numbers only',
-          },
-        ])
+      describe('for an existing entry', () => {
```

Comment:
`for a participation with existing details`? Given the `entry` is a participation and will always exist when we get to this page, whether or not details have been entered